docs/deploy.rst

```diff
@@ -31,11 +30,13 @@ local configuration or environment variables. See: `AWS Config`_.
 
 .. code-block:: console
 
+ $ export $PROJECT=...
+ $ export $ENV_TYPE=...
  $ aws cloudformation deploy \
         --template-file configuration/exodus-storage.yaml \
         --stack-name ... \
         --capabilities ... \
-        --parameter-overrides env=... project=...
+        --parameter-overrides env=$ENV_TYPE project=$PROJECT oai=...
```

Comment:
Not a big deal but I don't think we needed this change.
The environment variables before `exodus-lambda-pkg.yaml` are needed for the `build-package` script, so it makes sense to reuse them for the `cloudformation deploy` command.
Sorry for the confusion before; I missed that the environment variables were defined when I suggested we take them out of the `cloudformation deploy` command.
If it helps, I'll add some suggestions to illustrate what I have in mind.